src/main/java/net/minecraftforge/client/model/QuadTransformers.java

```diff
@@ -95,7 +95,19 @@ public static IQuadTransformer applyingLightmap(int lightmap)
     }
 
     /**
-     * {@return a {@link BakedQuad} transformer that sets the lightmap to the given emissivity (0-15)}
+     * @return A new {@link BakedQuad} transformer that applies the specified block and sky light values.
+     */
+    public static IQuadTransformer applyingLightmap(int blockLight, int skyLight)
+    {
+        return quad -> {
+            var vertices = quad.getVertices();
+            for (int i = 0; i < 4; i++)
+                vertices[i * IQuadTransformer.STRIDE + IQuadTransformer.UV2] = LightTexture.pack(blockLight, skyLight);
```

Comment:
Packing can be done once and then stored in a field. Even better: ack the parameters and then invoke the packed variant